core/controllers/topic_editor.py

```diff
@@ -383,9 +357,6 @@ def put(self, topic_id, assignee_id):
         """
         topic_domain.Topic.require_valid_topic_id(topic_id)
 
-        if assignee_id is None:
```

Comment:
Ditto, don't remove this.

Reply:
How can I forcibly assign assignee_id to be None? None or '' can not be passed in the url, we will get a 401 error as the url won't exist. Hence again this condition is un-testable.

Reply:
Not convinced this is true, what if you don't include assignee _id in the payload?

Reply:
If we don't include assignee_id, we will be getting an exception from webapp as there is no controller defined for that URL(the same happens for assignee_id = ''). We also cannot concatenate None to the URL, will be getting exception "cannot concatenate 'str' and 'NoneType' objects" from Python.

Reply:
But from the frontend, what if NULL is passed in as parameter, won't that be considered as None by python? If so, this check is required.

Reply:
Hmm then any idea how this can be tested in the backend?

Reply:
Not sure, you could manually check the url generated when `assignee_id`` is set as null from the frontend, and try replicating that in a test. If that doesn't work, I'm not sure.

Reply:
Actually, I was just checking the frontend to see where this was implemented but it seems the URL for this class, although declared as a constant, is never called and this functionality is actually done using a separate class in admin.py.
So, can you  remove this class and all its tests?

Reply:
Done